pkg/patterns/declarative/reconciler.go

```diff
@@ -65,7 +65,7 @@ type Reconciler struct {
 }
 
 type kubectlClient interface {
-	Apply(ctx context.Context, namespace string, manifest string, validate bool, args ...string) error
+	applier.Applier
```

Comment:
This seems reasonable.  I do wonder if there's any reason to keep the `kubectlClient` interface, if we have the exported applier.Applier interface...

Reply:
Agree with your comment, we can use `Applier` interface instead of `kubectlClient` interface.